python/foglamp/services/south/ingest.py

```diff
@@ -340,23 +353,15 @@ def increment_discarded_readings(cls):
             # Perform insert. Retry when fails.
             while True:
                 try:
-                    payload = dict()
-                    payload['readings'] = copy.deepcopy(readings_list)
-                    batch_size = len(payload['readings'])
+                    batch_size = len(readings_list)
```

Comment:
Again, We cannot have the batch size set to len(readings_list) on a low end device like raspberry pi. The Host machine will starve memory in that case. I suggest to keep in configurable

Reply:
I do not understand this comment. Since beginning, the batch_size is being calculated on reading_list count as batch_size is exactly this quantity only. What makes you think it is different now?